src/index.css

```diff
@@ -1,4 +1,19 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
-@import url("https://fonts.googleapis.com/css2?family=Oswald&display=swap");
```

Comment:
Why don't we just use import font instead of downloading the font?

Reply:
There is not much difference of using either. But it would be a better performance and reliability in the long run if we download the font (especially that font will be used widely).